test/compilable/libpath.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+TEMP_LIB_PATH=${RESULTS_DIR}/compilable/templibpath
```

Comment:
You need back slashes for optlink unconditionally. The test log also shows that adding a quote after the trailing back slash escapes the quote, so the trailing backslash needs to be moved after the quote.